docs/addons/container-insights.md

```diff
@@ -21,3 +28,121 @@ new EksBlueprint(app, 'my-stack-name', addOns, [], {
+
+NAME                                DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/cloudwatch-agent     1         1         1       1            1           <none>          107s
+daemonset.apps/fluentd-cloudwatch   1         1         1       1            1           <none>          106s
+```
+
+To enable or disable control plane logs with the console, please run the following command in your terminal - `aws eks update-cluster-config \
+    --region us-east-2 \
+    --name east-dev \
+    --logging '{"clusterLogging":[{"types":["api","audit","authenticator","controllerManager","scheduler"],"enabled":true}]}'`
+
+You should see a similar output as the following
+```json
+{
+    "update": {
+        "id": "<883405c8-65c6-4758-8cee-2a7c1340a6d9>",
+        "status": "InProgress",
+        "type": "LoggingUpdate",
+        "params": [
+            {
+                "type": "ClusterLogging",
+                "value": "{\"clusterLogging\":[{\"types\":[\"api\",\"audit\",\"authenticator\",\"controllerManager\",\"scheduler\"],\"enabled\":true}]}"
+            }
+        ],
+        "createdAt": 1553271814.684,
+        "errors": []
+    }
+}
+```
+
+You can also monitor the status of your log configuration update to your cluster by running the following command - `aws eks describe-update \
+    --region <region-code>\
+    --name <prod> \
+    --update-id <883405c8-65c6-4758-8cee-2a7c1340a6d9>`
+
+Once the update is complete, you should see a similar output 
+
+```json
+{
+    "update": {
+        "id": "<883405c8-65c6-4758-8cee-2a7c1340a6d9>",
+        "status": "Successful",
+        "type": "LoggingUpdate",
+        "params": [
+            {
+                "type": "ClusterLogging",
+                "value": "{\"clusterLogging\":[{\"types\":[\"api\",\"audit\",\"authenticator\",\"controllerManager\",\"scheduler\"],\"enabled\":true}]}"
+            }
+        ],
+        "createdAt": 1553271814.684,
+        "errors": []
+    }
+}
+```
+
+## View cluster level logs
+After you have enabled any of the control plane log types for your Amazon EKS cluster, you can view them on the CloudWatch console.
+
+To view these logs on the CloudWatch console follow these steps:
+
+1. Open the CloudWatch console and choose the cluster that you want to view logs for. The log group name format is /aws/eks/<cluster-name>/cluster.
+2. Choose the log stream to view. The following list describes the log stream name format for each log type.
+   1. Kubernetes API server component logs (api) – kube-apiserver-<nnn...>
+   2. Audit (audit) – kube-apiserver-audit-<nnn...>
+   3. Authenticator (authenticator) – authenticator-<nnn...>
+   4. Controller manager (controllerManager) – kube-controller-manager-<nnn...>
+   5. Scheduler (scheduler) – kube-scheduler-<nnn...>
+
+Next in the console, click on Log groups under Logs. 
+
+You will see under log streams all the log streams from your Amazon EKS control plane. 
+
+(insert image)
```

Comment:
Let's get the image and reference it here.

Reply:
Added